frontend/src/components/pages/NewDataMapper.js

```diff
@@ -104,13 +104,15 @@ export default ({ gateway, goToDataMappers }) => {
     if (isFormValid) {
       setFormState("loading");
       try {
+        const format = selectedTable.format;
```

Comment:
nit: don't need declare this const since it's only used once

Reply:
done